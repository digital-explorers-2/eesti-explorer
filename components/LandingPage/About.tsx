import React from 'react'
import MediumButton from '../MediumButton'
import Image from 'next/image'
import AboutImage from '../images/LandingPage/about.png'

export default function About() {
  return (
    <section className="w-full px-24 pt-14 flex justify-between" id="home">
        <div id="words_left" className="w-[50%] pt-3">
            <div>
                <p className="text-[#F57906]">Why choose us?</p>
                <h1 className="font-bold text-3xl leading-relaxed">Choose Eestie Explorer To Experience Estonia</h1>
            </div>
            <div className="pr-10">
                <p className="text-sm leading-loose pt-3">
                    Discover Estonia like never before with our user-friendly app 
                    that connects you to local guides and customizable tours 
                    tailored to your interests. We pride ourselves on offering 
                    a seamless booking process and expert guidance.
                </p>
            </div>
            <div className="pt-5">
                <MediumButton ><a href="/about">Learn more</a></MediumButton>
            </div>
            <div className='pt-10 flex gap-24'>
                <div>
                    <h1 className='text-3xl font-medium'>2</h1>
                    <p className='text-gray-500 pt-3'>Years<br/>Experience</p>
                </div>
                <div>
                    <h1 className='text-3xl font-medium'>20+</h1>
                    <p className='text-gray-500 pt-3'>Company<br/>Collaborations</p>
                </div>
                <div>
                    <h1 className='text-3xl font-medium'>67+</h1>
                    <p className='text-gray-500 pt-3'>Satisfied<br/>Customers</p>
                </div>
            </div>
        </div>
        <div className="w-[35%]">
            <Image src={AboutImage} className='flex float-right rounded-3xl' style={{boxShadow: '-10px 10px 20px rgba(0, 0, 0, 0.15)'}} alt="snow"/>
        </div>
    </section>
  )
}
