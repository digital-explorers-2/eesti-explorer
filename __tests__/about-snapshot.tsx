import About from "@/components/LandingPage/About"
import { render } from "@testing-library/react";

describe("About page snapshot unchanged", () => {
  it("renders about page", () => {
    const { container } = render(<About/>);
    expect(container).toMatchSnapshot();
  });
});