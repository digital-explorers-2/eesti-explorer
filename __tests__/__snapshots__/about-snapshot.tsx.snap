// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page snapshot unchanged renders about page 1`] = `
<div>
  <section
    class="w-full px-24 pt-14 flex justify-between"
    id="home"
  >
    <div
      class="w-[50%] pt-3"
      id="words_left"
    >
      <div>
        <p
          class="text-[#F57906]"
        >
          Why choose us?
        </p>
        <h1
          class="font-bold text-3xl leading-relaxed"
        >
          Choose Eestie Explorer To Experience Estonia
        </h1>
      </div>
      <div
        class="pr-10"
      >
        <p
          class="text-sm leading-loose pt-3"
        >
          Discover Estonia like never before with our user-friendly app that connects you to local guides and customizable tours tailored to your interests. We pride ourselves on offering a seamless booking process and expert guidance.
        </p>
      </div>
      <div
        class="pt-5"
      >
        <button
          class="text-sm bg-[#F57906] px-7 py-3 rounded-[7px] text-white font-semibold"
        >
          <a
            href="/about"
          >
            Learn more
          </a>
        </button>
      </div>
      <div
        class="pt-10 flex gap-24"
      >
        <div>
          <h1
            class="text-3xl font-medium"
          >
            2
          </h1>
          <p
            class="text-gray-500 pt-3"
          >
            Years
            <br />
            Experience
          </p>
        </div>
        <div>
          <h1
            class="text-3xl font-medium"
          >
            20+
          </h1>
          <p
            class="text-gray-500 pt-3"
          >
            Company
            <br />
            Collaborations
          </p>
        </div>
        <div>
          <h1
            class="text-3xl font-medium"
          >
            67+
          </h1>
          <p
            class="text-gray-500 pt-3"
          >
            Satisfied
            <br />
            Customers
          </p>
        </div>
      </div>
    </div>
    <div
      class="w-[35%]"
    >
      <img
        alt="snow"
        class="flex float-right rounded-3xl"
        data-nimg="1"
        decoding="async"
        height="40"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
        style="color: transparent; box-shadow: -10px 10px 20px rgba(0, 0, 0, 0.15);"
        width="40"
      />
    </div>
  </section>
</div>
`;
